### PR TITLE
Fix ContentTypeId conflict

### PR DESCRIPTION
**Pull request by CrazyHZM**
## What is the purpose of the change

Fix Serialization ContentTypeId conflict between avro protocol and protocoluff protocol #3926

## Brief changelog

XXXXX

## Verifying this change

XXXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests=false` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java [hunk 1]
```diff
@@ -875,4 +875,18 @@
     public static final String METRICS_PROTOCOL = "metrics.protocol";
 
 
+    /**
+     * Serizlization ContentTypeId
+     */
+    public static final byte AVRO_SERIALIZATION_ID = 1;
```
Comment (beiwei30) [inline dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java]: this is wrong, avro is 10...

Comment (beiwei30) [reply dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java]: *public static final byte AVRO_SERIALIZATION_ID = 11* should be better

Comment (CrazyHZM) [reply dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java]: > _public static final byte AVRO_SERIALIZATION_ID = 11_ should be better

What are the uses of 1 and 5 here?

Comment (beiwei30) [reply dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java]: let's simply leave it as is. I believe the original intention is: 1, 2, 4, 8, 16...

Comment (CrazyHZM) [reply dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java]: I have modified it.

File: dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java [hunk 2]
```diff
@@ -875,4 +875,18 @@
...
+    public static final byte JAVA_SERIALIZATION_ID = 3;
+    public static final byte COMPACTED_JAVA_SERIALIZATION_ID = 4;
+    public static final byte FASTJSON_SERIALIZATION_ID = 6;
+    public static final byte NATIVE_JAVA_SERIALIZATION_ID = 7;
+    public static final byte KRYO_SERIALIZATION_ID = 8;
+    public static final byte FST_SERIALIZATION_ID = 9;
+    public static final byte PROTOSTUFF_SERIALIZATION_ID = 10;
+    public static final byte AVRO_SERIALIZATION_ID = 11;
+    public static final byte GSON_SERIALIZATION_ID = 16;
+
```
Comment (cvictory) [inline dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java]: Maybe you can define a sub-constant class. What do you think?

Comment (satansk) [reply dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java]: +1 for a specific constants class

Comment (CrazyHZM) [reply dubbo-common/src/main/java/org/apache/dubbo/common/Constants.java]: I think this suggestion is good, I will modify it later.

**Pull request merged**